docs/release-source/release/spies.md

```diff
@@ -390,11 +390,16 @@ Array of return values, `spy.returnValues[0]` is the return value of the first c
 If the call did not explicitly return a value, the value at the call's location in `.returnValues` will be `undefined`.
 
 
-#### `spy.reset()`
+#### `spy.resetHistory()`
 
 Resets the state of a spy.
 
 
+#### `spy.reset()`
```

Comment:
Remove this. If it's deprecated we don't need to spread knowledge of its existence. Only old code-bases should use this.